app/components/transactions/LatestTransactions.tsx

```diff
@@ -115,22 +115,28 @@ const LatestTransactions = ({ navigateToAllTransactions }: Props) => {
   };
 
   const renderTransaction = (tx: TxView) => {
-    const { id, status, amount, sender, timestamp, senderNickname } = tx;
-    const isSent = sender === getAddress(publicKey);
+    const { id, status, timestamp, contacts, meta } = tx;
+    const isSent = tx.method === 1 && !!tx.payload?.Arguments?.Destination;
```

Comment:
can we tx.method === 1 , transfer 1 to some constant value if it possible , to understand what tx method 1 means

Reply:
Actually, this is a temporary solution.
When we'll have other account types we'd have to replace this solution with something better :)
Anyway, I've added enum there ;)